mule-extensions-api/src/test/java/org/mule/runtime/extension/internal/loader/FlatExtensionModelFactoryTestCase.java

```diff
@@ -415,7 +421,10 @@ public void connectionDefaultStereotype() {
 
     ExtensionModel extensionModel = load();
 
-    assertThat(extensionModel.getConnectionProviderModel("myConnection").get().getStereotype(), is(CONNECTION));
+    StereotypeModel connectionStereotype = extensionModel.getConnectionProviderModel("myConnection").get().getStereotype();
+    assertThat(connectionStereotype.getType(), equalTo("MY_CONNECTION"));
+    assertThat(connectionStereotype.getNamespace(), equalTo(NAMESPACE));
+    assertThat(connectionStereotype.getParent().get(), is(CONNECTION));
```

Comment:
should we add here tests for the other components which also support stereotypes, i.e. source, operation?

Reply:
added